conftest.py

```diff
@@ -227,6 +227,11 @@ def sr_disk(request, host):
             f"disk or block device {disk} is either not present or already used on master host"
     yield disk
 
+@pytest.fixture(scope="session")
+def sr_disk_devid(host, sr_disk):
+    # [5:] remove /dev/ at is not expected by normal sr_disk fixture
```

Comment:
You don't use `[5:]` but `[len('/dev/')]`. ;)
```suggestion
    # Remove /dev/ at is not expected by normal sr_disk fixture
```

Reply:
Also, your fixture herits from `sr_disk` and not the other way arount so I don't understand why what's expected by the parent fixture matters here.